### PR TITLE
feat(divider): add divider component

### PR DESCRIPTION
**Pull request by xiaojiabin**
feat(divider): add divider component

### CODE REVIEW

**Comment by xiaojiabin** [comment]
its my first time to commit, you can give me some suggestions ? thanks

File: test/unit/specs/divider.spec.ts [hunk 1]
```diff
@@ -0,0 +1,31 @@
+import { mountWithElement } from '../util'
+
+describe('Divider', () => {
+  let vm
```
Comment (a631807682) [inline test/unit/specs/divider.spec.ts:4]: Should declare the type.

File: packages/divider/src/divider.vue [hunk 2]
```diff
@@ -0,0 +1,34 @@
+<template function>
+  <div v-bind="$attrs" :class="['el-divider', `el-divider--${direction}`]">
```
Comment (a631807682) [inline packages/divider/src/divider.vue:2]: `v-bind="$attrs"` dose not need in vue-next. See [active-rfcs/0031-attr-fallthrough.md](https://github.com/vuejs/rfcs/blob/master/active-rfcs/0031-attr-fallthrough.md)

`<template function>` dose `function` necessary?

**Comment by xiaojiabin** [comment]
不知道为什么覆盖率一直不能通过

**Comment by a631807682** [comment]
覆盖率那个无所谓，权限问题吧，我没查。

**Pull request merged**